src/Config/ActionComponent/ConfigDefinition.php

```diff
@@ -18,8 +18,10 @@ public function getParametersDefinition(): ArrayNodeDefinition
             ->children()
                 ->arrayNode('db')
                     ->isRequired()
+                    ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('host')->isRequired()->cannotBeEmpty()->end()
+                        ->integerNode('port')->defaultValue(1025)->end()
```

Comment:
trochu mě mate, že to je na dvou místech (a v druhém navíc není `addDefaultsIfNotSet()`)

Reply:
Smrsknu to